docs/guides/advanced-use-case/cross_cluster_backup.md

```diff
@@ -0,0 +1,582 @@
+---
+title: Cross-Cluster Backup and Restore | Stash
+description: A guide on how to use backup and restore across clusters using Stash.
+menu:
+  docs_{{ .version }}:
+    identifier: advance-use-case-cross-cluster-backup
+    name: Cross-Cluster Backup and Restore
+    parent: advance-use-case
+    weight: 20
+product_name: stash
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+# Cross-Namespace Backup and Restore
+
+This guide will show you how to take backup and restore across clusters using Stash.
+
```

Comment:
- Before You Begin
- Backup from `prod` Cluster
  - Prepare cluster
  - Deploy workload
  - Prepare backend
  - Configure backup
  - Verify backup
- Restore into `staging` Cluster
  - Prepare cluster
  - Deploy recovered workload
  - Prepare backend info
  - Configure restore
  - Verify restored data
- Cleanup
  - Cleanup `prod` cluster
  - Cleanup `staging` cluster